CommunityProject/Assets/_Game/Scripts/Runtime/Audio/BackgroundMusicPlayer.cs

```diff
@@ -0,0 +1,55 @@
+using Cysharp.Threading.Tasks;
+using System;
+using UnityEngine;
+
+namespace BoundfoxStudios.CommunityProject.Audio
+{
+	/// <summary>
+	/// This component plays <see cref="AudioClip"/>s from a given <see cref="PlaylistSO"/> on the <see cref="AudioSource"/> on same GameObject.
+	/// </summary>
+	[RequireComponent(typeof(AudioSource))]
+	public class BackgroundMusicPlayer : MonoBehaviour
+	{
+		[SerializeField]
+		private PlaylistSO playlist;
```

Comment:
Der Vollständigkeit halber: 

```suggestion
		private PlaylistSO Playlist;
```